src/haxelib/client/Main.hx

```diff
@@ -1344,6 +1345,25 @@ class Main {
 		}
 	}
 
+	function pathonly() {
```

Comment:
OK, this is cheating. 😃 In the interest of not duplicating bugs, do you think that we can combine the contents of the two commands into one shared routine?

Reply:
OK @EricBishton @kLabz I didn't want to do it at first, since it'd require a `.bind` and would make reading the method harder. But seems other methods also use binds - so I've merged them now.

Reply:
@EricBishton the PR has now shifted to a separate method, so there's now a fully separate `libpath()` method, to not interfere with the `path` logic ( as both have different objectives ).